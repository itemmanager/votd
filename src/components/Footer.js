import styled from "styled-components";
import React from "react";

const StyledFooter = styled.footer`
  background: black;
  display: flex;

  a {
    font-size: 18px;
    color: white;
    padding: 1em;
  }
`

export function Footer() {
    return <StyledFooter>
        <a href="https://itemmanager.uk/" target="_blank" rel="noreferrer">Item Manager</a>
    </StyledFooter>
}
