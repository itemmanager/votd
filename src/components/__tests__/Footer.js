import renderer from 'react-test-renderer';
import { Footer } from "../Footer";
import 'jest-styled-components'


test('Footer snapshot', () => {
    const component = renderer.create(
        <Footer />
    )
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})
