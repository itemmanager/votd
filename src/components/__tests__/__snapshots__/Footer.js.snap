// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
.c0 {
  background: black;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c0 a {
  font-size: 18px;
  color: white;
  padding: 1em;
}

<footer
  className="c0"
>
  <a
    href="https://itemmanager.uk/"
    rel="noreferrer"
    target="_blank"
  >
    Item Manager
  </a>
</footer>
`;
